### ISSUE TITLE
Request for a standardized DROID training recipe for fair RoboLab comparisons

### ISSUE THREAD

**Issue by Vincia-Jun** [NVlabs/RoboLab#21]
Hi RoboLab team,

Thank you for releasing RoboLab. We are interested in evaluating our VLA model on this benchmark and comparing it fairly with the reported baselines.

Our understanding is that the main experimental loop is to train or fine-tune policies on real-world DROID data and evaluate them in RoboLab simulation. However, we could not find a complete, standardized training protocol for reproducing the baseline policies or for training new models under a comparable data and compute budget.

For example, RoboLab provides the `pi05_droid_jointpos` evaluation checkpoint and inference configuration, while the OpenPI repository provides a `pi05_full_droid_finetune` recipe described as an approximate open-source reproduction of the π0.5-DROID training pipeline. The latter uses 100k optimization steps, a global batch size of 256, approximately one epoch on DROID v1.0.1, joint-position actions, and idle-transition filtering. It is unclear whether this recipe exactly matches the checkpoint evaluated in RoboLab.

Could you please clarify or release a recommended training protocol for fair comparison? In particular, it would be very helpful to specify the following.

### 1. Training data

* Exact DROID version and data subset
* Whether all approximately 75k language-annotated episodes are used
* Train/validation split
* Whether only successful trajectories are used
* Any excluded trajectories or sites
* Idle-action filtering and other sampling filters

### 2. Training budget

* Number of optimization steps or epochs
* Global batch size
* Number and type of GPUs
* Gradient accumulation, if any
* Optimizer and learning-rate schedule
* Weight decay, EMA, and training precision
* Whether training budget should be matched by optimizer steps, sampled windows, epochs, or compute

### 3. Policy and action configuration

* Base checkpoint used for each reported baseline
* Joint-position versus joint-velocity action targets
* Action horizon or chunk length during training
* Action horizon or execution horizon during evaluation
* State and action normalization statistics
* Joint ordering and gripper encoding
* Image observations and camera selection
* Input resolution and image augmentation

### 4. Relationship to the released π0.5 checkpoint

Could you also clarify whether the RoboLab checkpoint

```text
gs://openpi-assets-simeval/pi05_droid_jointpos
```

was trained using the currently released OpenPI `pi05_full_droid_finetune` configuration, or whether there are additional differences in the data pipeline, action representation, filtering, hyperparameters, or training duration?

We noticed, for example, that the public full-DROID training recipe uses an action horizon of 16, while the RoboLab inference configuration for `pi05_droid_jointpos` appears to use an action horizon of 15. It would be useful to know whether this reflects a training/evaluation distinction or a different checkpoint configuration.

### 5. Benchmark comparison policy

For models with different architectures or pretraining sources, could you recommend how results should be reported?

In particular:

* Are additional robot pretraining datasets allowed?
* Is general-purpose VLM or VLA pretraining allowed?
* Should a DROID-only result be reported separately?
* Should all models use an identical DROID training subset?
* Should training compute or the number of sampled DROID windows be controlled?
* Is there a recommended metadata or configuration template for benchmark submissions?

A common training recipe, or at least a clearly defined data and compute budget, would make comparisons across different model architectures substantially more interpretable and reproducible.

Related resources:

* RoboLab repository: https://github.com/NVLabs/RoboLab
* RoboLab paper: https://arxiv.org/abs/2604.09860
* OpenPI DROID training guide: https://github.com/Physical-Intelligence/openpi/blob/main/examples/droid/README_train.md

Thank you for your help and for releasing the benchmark.

**Comment by xuningy** [comment]
Hi, all checkpoints and policy clients featured are directly obtained off-the-shelf. pi-family models are obtained directly from pi, e.g., `gs://openpi-assets-simeval/pi05_droid_jointpos`. Certain models have made their training recipe publicly available for DROID, for example: [pi0.5](https://github.com/Physical-Intelligence/openpi/blob/main/examples/droid/README_train.md) and [Cosmos3-Nano-Policy](https://github.com/NVIDIA/cosmos/blob/main/cookbooks/cosmos3/generator/action/finetune/README.md). You may submit policies to be featured on the leaderboard [here](https://docs.google.com/forms/d/e/1FAIpQLSe79wVRzup75p08qMK58PKg9eKhBEK6Et7gIE8ioV9e6Gd1kA/viewform).
